File: python/scripts/bulk_update_cost_centers.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
response = travelperk.cost_centers().cost_centers().bulk_update().set_ids(
    ["1", "2"]
).set_archive(True).save()
write_output("bulk_update_cost_center", [
    response.updated_count
])
travelperk.cost_centers().cost_centers().bulk_update().set_ids(
    ["1", "2"]
).set_archive(False).save()

File: php/scripts/bulk_update_cost_centers.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$response = $travelperk->costCenters()->costCenters()->bulkUpdate()->setIds(["1", "2"])
	->setArchive(True)->save();
write_output("bulk_update_cost_center", [
	$response->updatedCount,
]);
$response = $travelperk->costCenters()->costCenters()->bulkUpdate()->setIds(["1", "2"])
	->setArchive(False)->save();
